cluster-controller/src/main/java/io/strimzi/controller/cluster/model/KafkaCluster.java

```diff
@@ -64,12 +64,16 @@
     public static final String KEY_METRICS_CONFIG = "kafka-metrics-config";
     public static final String KEY_STORAGE = "kafka-storage";
 
+    public static final String KEY_JVM_OPTIONS = "kafka-jvmOptions";
+    public static final String KEY_RESOURCES = "kafka-resources";
+
     // Kafka configuration keys
     public static final String KEY_KAFKA_ZOOKEEPER_CONNECT = "KAFKA_ZOOKEEPER_CONNECT";
     public static final String KEY_KAFKA_DEFAULT_REPLICATION_FACTOR = "KAFKA_DEFAULT_REPLICATION_FACTOR";
     public static final String KEY_KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR = "KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR";
     public static final String KEY_KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR = "KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR";
     private static final String KEY_KAFKA_METRICS_ENABLED = "KAFKA_METRICS_ENABLED";
+    public static final String KAFKA_HEAP_OPTS = "KAFKA_HEAP_OPTS";
```

Comment:
I think it's a mistake that these things are prefixed by `KEY_`. Right now there are the keys in the config map and the environment variables, which are basically two distinct sets, but they both use the `KEY_` prefix, which is confusing. I think it would be clearer if they were prefixed by `ENVVAR_`. Wdyt?

If you agree I'll switch to `KEY_` for now, but open a PR to use the new prefix.

Reply:
Yeah, sounds good. I was confused by this the other day as well.